tests/e2e/backup_restore_suite_test.go

```diff
@@ -148,11 +148,11 @@ func runBackupAndRestore(brCase BackupRestoreCase, expectedErr error, updateLast
 	time.Sleep(brCase.AppReadyDelay)
 	// create backup
 	log.Printf("Creating backup %s for case %s", backupName, brCase.Name)
-	backup, err := CreateBackupForNamespaces(dpaCR.Client, namespace, backupName, []string{brCase.ApplicationNamespace}, brCase.BackupRestoreType == RESTIC || brCase.BackupRestoreType == KOPIA, brCase.BackupRestoreType == CSIDataMover)
+	backup, err := CreateBackupForNamespaces(dpaCR.Client, namespace, backupName, []string{brCase.ApplicationNamespace}, brCase.BackupRestoreType == RESTIC || brCase.BackupRestoreType == KOPIA, brCase.BackupRestoreType == CSIDataMover, "180s")
```

Comment:
we use `time.<unit>` in a lot of places, why use a string here?

Reply:
To easy write tests and align with what server flags are expected in upstream velero. Also align with the velero crd which is string:

https://github.com/vmware-tanzu/velero/pull/5104/files#diff-1d9e81ef7720633f549a8239f569a20ebd656c9d880be086d44e18954a035cc8R44

Reply:
change to `3*time.Minute`, delete parse function and use `String()` method from `time` package